tests/golden/capacity-alerts-with-node-labels/openshift4-monitoring/openshift4-monitoring/10_configmap_user_workload.yaml

```diff
@@ -14,6 +14,7 @@ data:
     "prometheus":
       "nodeSelector":
         "node-role.kubernetes.io/infra": ""
+      "remoteWrite": []
```

Comment:
This introduces a small noop diff for every catalog. We could also remove this, but then the patching for the user workload monitoring would be slightly different from the cluster monitoring patch. 

So I opted for the noop diff, but I can also change it so there is no diff if you prefer that.